frontend/packages/component-library/src/divider/Divider.tsx

```diff
@@ -35,8 +35,8 @@ const Divider: React.FC<DividerProps> = ({
   <hr
     className={classNames(
       moduleStyles.divider,
-      moduleStyles[`divider-${color}`],
-      moduleStyles[`divider-${margin}`],
+      moduleStyles[`divider-color-${color}`],
```

Comment:
As I mentioned in [this comment](https://github.com/code-dot-org/code-dot-org/pull/63829#discussion_r1954309125) - sorry for misunderstanding, for color I think it may also be ok to keep old color classname approach.
Have some doubts though, so don't have strong opinion here on both

Reply:
Closing the loop: we talked offline about using this norm and ticket is here to update existing DSCO components: [CMS-352](https://codedotorg.atlassian.net/browse/CMS-352)